src/cata2data/cata2data.py

```diff
@@ -32,6 +32,7 @@ def __init__(
         transform: Optional[Callable] = None,
         catalogue_preprocessing: Optional[Callable] = None,
         wcs_preprocessing: Optional[Callable] = None,
+        image_preprocessing: Optional[Callable] = None,
```

Comment:
I am not sure this does anything that isn't already provided by `transform`.

Reply:
Maybe - but at the moment I think it's important for backwards compatibility

Reply:
What about a scenario where this is caught in kwargs?

i.e. something like:
```suggestion
    **kwargs)
      
    ...
    self.kwargs = kwargs

    if "imgae_processing" in kwargs:
        image = self.kwargs["image_processing"](image)
          
```

This will allow the class to be constructed with it, but without confusing new users as to what the difference might be?

Reply:
yes this would also work

Reply:
Ok. Ill make the change and approve.